### ISSUE TITLE
📚 Documentation:  Very Low color contrast between sidebar text and background (ui/ux)

### ISSUE THREAD

**Issue by manish-basargekar** [appwrite/appwrite#2941]
### 💭 Description

Colors (#1E849E & #F02E65) in the light theme fails to achieve a minimum contrast ratio of 4.5:1.
Also, sidebar text has a font-size of 13px compared to body (16px), which makes it  a little difficult to read.
Using a dark shade of (#1E849E) would fix the sidebar issue.
I used this tool to check contrast [https://wave.webaim.org/](https://wave.webaim.org/)

![image](https://user-images.githubusercontent.com/64839201/158042637-d7dcb839-0006-45a2-a56c-05935e385c9f.png)

![image](https://user-images.githubusercontent.com/64839201/158042961-892bb56b-f816-4a2e-87ff-0b267b4cb762.png)

![image](https://user-images.githubusercontent.com/64839201/158043571-c2b00308-c92b-4f14-866f-896f1ae635a6.png)

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by eldadfux** [comment]
We are working right now on improving the entire accessibility of both the Appwrite console and the homepage. We aim to comply with at least level A to begin with.

https://www.levelaccess.com/accessibility-regulations/accessibility-standards/#:~:text=Level%20A%20%E2%80%93%20This%20level%20defines,the%20minimum%20set%20of%20requirements.

**Comment by eldadfux** [comment]
We have a completely new design for the website and docs that is following accessibility standard.
